mlctl/clis/template/{{cookiecutter.package_name}}/model/train.py

```diff
@@ -0,0 +1,17 @@
+import uuid
+import os
+import shutil
+
+
+def train():
+    print('Hello, put your model training code here')
+
+
+def main(ta):
+    """
+    Load features and labels, train the tree, and serialize model artifact.
+    Note: This is the training entrypoint used by baklava!
+    """
```

Comment:
I think there should be a `train()` call within main

Reply:
correction, based on the setup.py, the `train()` is called. In that case, the training adapter should be embedded there